advanced/Scripts/webpage.sh

```diff
@@ -438,6 +438,19 @@ ra-param=*,0,0
     fi
 }
 
+EnableDHCPSeeded() {
+  case "${args[2]}" in
+    "-s")
+      source "${setupVars}";
+      change_setting "DHCP_ACTIVE" "true";
+      echo EnableDHCP blah "$DHCP_ACTIVE" "$DHCP_START" "$DHCP_END" "$DHCP_ROUTER" "$DHCP_LEASETIME" "$PIHOLE_DOMAIN" "$DHCP_IPv6" "$DHCP_rapid_commit";
```

Comment:
Is `blah` supposed to be here?

Reply:
Dangit. Of course not. :-D
Do you need another PR?

Reply:
It's allowed me to change the base on this end cleanly, so feel free to push to the same PR. Even better, rebase on `development` and force push